docs/orchestration/rbac/overview.md

```diff
@@ -0,0 +1,88 @@
+# Role Based Access Controls <Badge text="Cloud"/>
+
+Prefect Cloud Enterprise plans come with customizable role based access controls.
+
+**Roles** are used to denote the permissions a user has as part of a team.
+
+Roles can be assigned to both users and service accounts.
+
+### Default Roles
+
+Prefect Cloud contains three default roles
+
+- Administrator
+- User
+- Restricted User
+
+Default roles cannot be modified.
+
+#### Administrators
+
+Administrators have all permissions given to users, but are also able to actively manage their team in Prefect Cloud. This means they are able to invite new members, remove unwanted members, and change the roles of other members of their team.
+
+#### Users
+
+Users have permission to perform all actions required for daily use of Prefect Cloud; this includes registering new flows, kicking off flow runs, etc. They do not have permission to perform actions that impact other members of the team, meaning they cannot change the roles of their teammates, invite others to their tenant, etc.
+
+#### Restricted Users
+
+Restricted users are able to view all pages as a normal user would, but are unable to take any actions in Prefect Cloud. For example, they cannot create projects, kick off flow runs, etc. In essence, restricted users are read-only members of a team.
+
+### Custom Roles
+
+To manage permissions more granularly, you can create new roles with custom sets of permissions. Custom roles are scoped to a team.
+
+When creating or updating a role, users cannot grant the role more permissions than they possess.
+
+Custom roles can be created, edited, and deleted on the [Roles page](https://cloud.prefect.io/team/roles).
+
+Custom roles can also be created, edited, and deleted via the GraphQL API.
+
+:::: tabs
+::: tab Create
```

Comment:
tab names for `Create`, `Update`, and `Delete` are deliberately shortened
when testing locally, tabs would show up as blank if the combined length of tab names was too long

Reply:
I'm not sure this is a good fit for tabs. I think of them as good for docs where there are several ways to do the same thing. Not a strong feeling though.

Reply:
fair enough, I'll break it out